### ISSUE TITLE
Path missing when uploading directory

### ISSUE THREAD

**Issue by pippo111** [react-dropzone/file-selector#5]
When using file selector fromEvent handler on `<input {..properties} type='file' webkitdirectory='true' />` full path is missing.

This is example from dropzone event `getDataTransferItems={e => fromEvent(e)}`:

```
File(106182)
  lastModified: 1538560178310
  lastModifiedDate: Wed Oct 03 2018 11:49:38 GMT+0200 (czas środkowoeuropejski letni) {}
  name : "audit-with-purge.png"
  size: 106182
  type: "image/png"
  webkitRelativePath: ""
  path: "/test/audit-with-purge.png"
```

And this is from input onChange event with fromEvent handler:
```
File(106182)
  lastModified: 1538560178310
  lastModifiedDate: Wed Oct 03 2018 11:49:38 GMT+0200 (czas środkowoeuropejski letni) {}
  name: "audit-with-purge.png"
  size: 106182
  type: "image/png"
  webkitRelativePath: ""
  path: "audit-with-purge.png"
```

Is there a way to preserve full path ?

EDIT:
In version 0.1.8 onChange actually adds `webkitRelativePath`, but path is still different and missing full path compared to one from `getDataTransferItems`

**Comment by rolandjitsu** [comment]
@pippo111 I've just released `v0.1.9` of [file-selector](https://github.com/react-dropzone/file-selector) which should use `{webkitRelativePath}` for `{path}`if it exists.

**Comment by pippo111** [comment]
Just checked it and it works as expected, thank you @rolandjitsu
